src/utils/setup.py

```diff
@@ -6,7 +6,7 @@
 import torch
 from src.dataclass import Context
 from src.model import LinearAttention
-from src.utils.formatting import syntax_print
+from src.utils.formatting import print
```

Comment:
_Deepsource will complain about this overriding a built-in function_

Reply:
If that's the case, we should rename formatting.print, correct?